TROUBLESHOOTING.md

```diff
@@ -11,6 +11,7 @@ This guide helps you diagnose and resolve common issues with the Azure MCP Serve
     - [Can I select what tools to load in the MCP server?](#can-i-select-what-tools-to-load-in-the-mcp-server)
     - [Why does VS Code only show a subset of tools available?](#why-does-vs-code-only-show-a-subset-of-tools-available)
```

Comment:
Similarly can you also update this line to begin with VS Code. This groups all 3 logically. 

Consider moving them into the Tool Limitation section below and also rename that to VS Code...

I think this is important so that users can quickly review or skip over all VS Code specific TSG entries.